app/controllers/course/experience_points_records_controller.rb

```diff
@@ -8,8 +8,13 @@ class Course::ExperiencePointsRecordsController < Course::ComponentController
   before_action :add_breadcrumbs
 
   def index # :nodoc:
+    updater_ids = @experience_points_records.active.pluck(:updater_id)
+    @course_user_preload_service =
+      Course::CourseUserPreloadService.new(updater_ids, current_course)
+
     @experience_points_records =
-      @experience_points_records.active.includes(:updater).order(updated_at: :desc).page(page_param)
+      @experience_points_records.active.includes { actable.assessment.lesson_plan_item }.
```

Comment:
This works for now because we have only one item that acts_as_experience_points_record. Will have to be rewritten when after we are done with surveys.